Some fixes

Kaladrius2trip opened the pull request:
For searching tags by IndexOf in existing binding files now uses Environment.NewLine. 
Because in Windows default newline symbols it's "\r\n" instead "\n". Maybe should use RegEx for this.

And all library build with prefix "lib" so... 
I have no idea how to fix it. I use CMake command to replace this by empty string.

Kaladrius2trip committed:
Removed prefix "lib" after build C++ library

Kaladrius2trip committed:
Fixed index of tags with Environment.NewLine "\r\n"

jacksondunstan commented:
@Kaladrius2trip Thanks for submitting this PR! I like the newline change and the updates to some of the Unity packages. May I ask why the "lib" prefix should be stripped off of build file names?

Kaladrius2trip committed:
Fix cmake lib prefix for mingw on win32 platform

Kaladrius2trip committed:
Replace all "\n" symbol by Environment.NewLine

Kaladrius2trip commented:
@jacksondunstan I use MinGW compiler. And library have this prefix on windows platform, but in const you are set lib name without prefix. I added condition for this in CMakeList. 

And i changed all "\n" to Environment.NewLine
When use StringBuilder.AppendLine it commonly use NewLine from Environment. With that might have some issue. But i cheked example and generated code looks like same.

jacksondunstan commented:
@Kaladrius2trip Thanks for explaining the "lib" prefix and for changing it to only be stripped for MinGW. Thanks also for the the change to output environment-specific newlines. I went ahead and merged your changes.
